API.md

```diff
@@ -1380,7 +1381,7 @@ On success, the server will return the newly created resource with an id. This i
 ### Updating
 An update request creates a new current version for an existing resource.
 
-**PLEASE NOTE:** This means that if certain attributes of the resource are omitted in the `PUT` requests, they will be replaced with `nil` values, as is expected with `PUT` requests. The Sara Alert team is planning on supporting `PATCH` requests in the future to support updates where only fields that should be changed need to be included.
+**PLEASE NOTE:** The API supports `PUT` and `PATCH` requests, which update an existing resource in different ways. A `PUT` request will replace the entire existing resource. This means that if certain attributes of the resource are omitted in the `PUT` requests, they will be replaced with `nil` values. A `PATCH` request will only modify the attributes indicated in the request, which must follow the [JSON Patch specification](https://tools.ietf.org/html/rfc6902). Omitted attributes are unchanged.
```

Comment:
This is great! I think you should also add a sentence in both sections which references the other method, e.g. for PUT: "NOTE: this is a PUT operation, it will replace the entire resource. If you intend to modify specific attributes instead, see PATCH"

Reply:
Updated the docs in https://github.com/SaraAlert/SaraAlert/pull/522/commits/b40fde9b0fbaa7a506f463b7b9bfc4dcef649c35.